GlotovES/HW5/task_2.py

```diff
@@ -0,0 +1,9 @@
+def print_fibon(n):
```

Comment:
I would recommend you use more meaningful name of variable
look at this code:
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])